packages/office-addin-telemetry/src/jsonData.ts

```diff
@@ -0,0 +1,65 @@
+
```

Comment:
maybe rename this file to telemetryJsonData.ts (to rename it, do 'git mv jsonData.ts telemetryJsonData.ts) and fix up imports accordingly